doc/tutorial.md

```diff
@@ -0,0 +1,175 @@
+# Getting Started with Submitting Job
+
+## Download and Configure paddlectl
+
+`paddlectl` is a command-line tool that submits distributed training job to the cloud.
```

Comment:
"submits distributed training job to the cloud" => "submits distributed training jobs to the paddle cloud"